skrobot/coordinates/base.py

```diff
@@ -1018,32 +1025,30 @@ def assoc(self, child, relative_coords=None,
                 'Please use `relative_coords` instead',
                 DeprecationWarning)
             relative_coords = kwargs['c']
+
+        is_invalid_assoc = (child.parent is not None) and (not force)
+        if is_invalid_assoc:
+            msg = "child already has a assoc relation with {0} \
```

Comment:
This error message has unnecessary space as follow.
Please fix it.
RuntimeError: child already has a assoc relation with #<CascadedCoords:a 0x7ff201deaa50 0.100 0.000 0.000 / 1.6 -1.0 0.0>                     to overwrite this, please specify force=True